pkg/services/tree/sync.go

```diff
@@ -19,7 +19,7 @@ import (
 var ErrNotInContainer = errors.New("node is not in container")
 
 // Synchronize tries to synchronize log starting from the last stored height.
-func (s *Service) Synchronize(ctx context.Context, cid cid.ID, treeID string) error {
+func (s *Service) Synchronize(ctx context.Context, cid cid.ID, treeID *string) error {
```

Comment:
Let's not complicate this function further. It has a clear purpose -- synchronize a single tree, the caller can list the trees and loop, it is not hard. If there are more callers, I would prefer a separate method/function.

Reply:
fixed. but it has become less readable IMO